### PR TITLE
publish on ci

### PR DESCRIPTION
**Pull request by charlierudolph**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,19 +4,17 @@ language: go
...
 git:
   depth: 2
 
 before_install:
-  # clone a real copy of the repo because Travis only clones one branch
-  # - git clone --depth 10 --no-single-branch https://github.com/$TRAVIS_REPO_SLUG.git $TRAVIS_REPO_SLUG
-  # - cd $TRAVIS_REPO_SLUG
-  # - git checkout $TRAVIS_COMMIT
-  - sudo add-apt-repository -y ppa:masterminds/glide && sudo apt-get update
-  - sudo apt-get install glide
```
Comment (charlierudolph) [inline .travis.yml]: Remove comments and we aren't using glide in travis

File: .travis.yml [hunk 2]
```diff
@@ -4,19 +4,17 @@ language: go
 
 go: 1.8
 
+env:
+  global:
+    secure: yS2NJpLP2fI537FSxvpyeEmUGxJgt22RfibJQRYCm7LdX0EoJ5VxSyp/eCkPbfgF+KaOywyEllrHaIa94P7Ks/zoAQ8oPVa9JOAYzNsGv2wnaer3C0NEFkx4ZBQp3oL2cBpFHdJBOWnlUQWCfXkgyKPYUKd8HlKN0C2DsCm1EzNYEva0Z8NaGj6UOW0XqwPb+0xtQkHKplhTrmKa9skCxe8Yn/EpwXb1LgsTQwk+kvM5s5OlTF7dQ9rYzc7nHOWDODWxkoNg12arK/jdmCO7bhR0VX6xL4VIU9H88yY7QODsbnCEAx322iqD4k/MHnfU0VjI5yXjIHYc3W+34oGTNGhwbpNwsT5pivXHFLDL/3zwxBJY7yym9nFVp+nqnBpj6jKbhHRuQbWKyCIV0sVgxXiXZvuqJNRHr2fA5UPdmgPxXnSyjx73VyQMCwO/mk+QTrtaUSKEZTc08rq4T8k74LzpsAJ7THDE9vlbarnAyyp2WSEY0AwngS9o4ixsMmHtaM4w3sKPXeuLdalCLIv+MYbzrlnUyLay8FXyAr7jdtZ96uMhLzFRCgAwr7vhFCJ+QEveN5Lb2L76XlhPp+nuuXuy62eZSdxAUXI0+FtztoYCER3WKIhbhiW+64yC2YanrNa5uqHRS+MUGFYBntNROPPUlBkYZ4wWYtGhPDMkwoo=
```
Comment (charlierudolph) [inline .travis.yml]: @alexdavid helped me setup a exosphere specific username/password for docker. @kevgo I used the npm auth token you gave me

Comment (kevgo) [inline .travis.yml]: Nice!

File: .travis.yml (hunk 1 above)
Comment (kevgo) [inline .travis.yml]: thanks for cleaning up this technical drift!

**Pull request merged**